specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-04-01-preview/machineLearningServices.json

```diff
@@ -1228,6 +1228,284 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/Microsoft.MachineLearningServices/workspaces/{workspaceName}/linkedWorkspaces": {
+      "get": {
+        "tags": [
+          "LinkedWorkspaces"
+        ],
+        "x-ms-examples": {
+          "ListLinkedWorkspaces": {
+            "$ref": "./examples/listLinkedWorkspaces.json"
+          }
+        },
+        "operationId": "LinkedWorkspaces_List",
+        "description": "List linked workspaces under a workspace per some query criteria.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "name": "link-subscriptionId",
```

Comment:
These query parameters look like filters. Our REST api guidelines mandate ODATA $filter syntax for this use case: https://github.com/microsoft/api-guidelines/blob/vNext/Guidelines.md#97-filtering

Are these query filters?

Reply:
yes, do you have some utility class to parse the filter in that format? otherwise I'd prefer remove the filters in private preview and add the formatted filter per the standard later.